### PR TITLE
Chat Cambrioleur - December Update

### PR DESCRIPTION
**Pull request by cgalvan**
Updates from December 2021 game jam:

- Added sleeping guard dog
- Added all remaining sounds and integrated with game events
- Prevent yarn ball collection sound from clashing with game ending sound
- Added chair to sleeping guard dog
- Re-enabled Credits and Quit options on main menu
- Updated the victory and failure end screens
- Added basic credits screen
- Added treasure chest

BasicCreditsScreen
GuardDawgSleepingOnChair

### CODE REVIEW

**Review by amzn-leenguy — APPROVED** [review]

File: Sounds/wwise/mainsoundbank.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	590716617	narrator-on-level-end	C:\ChatCambrioleur\Sounds\wwise_project\.cache\Windows\SFX\narrator-on-level-end_10C4C929.wem		\Actor-Mixer Hierarchy\Default Work Unit\on-level-success\narrator-on-level-end		1322968
+	764658650	narrator-on-level-fail	C:\ChatCambrioleur\Sounds\wwise_project\.cache\Windows\SFX\narrator-on-level-fail_10C4C929.wem		\Actor-Mixer Hierarchy\Default Work Unit\on-level-fail\narrator-on-level-fail		861952
+	864816229	catburglar-on-game-failure	C:\ChatCambrioleur\Sounds\wwise_project\.cache\Windows\SFX\catburglar-on-game-failure_10C4C929.wem		\Actor-Mixer Hierarchy\Default Work Unit\on-level-fail\catburglar-on-game-failure		853480
+	908417878	catburglar-on-mission-start	C:\ChatCambrioleur\Sounds\wwise_project\.cache\Windows\SFX\catburglar-on-mission-start_10C4C929.wem		\Actor-Mixer Hierarchy\Default Work Unit\catburglar-on-mission-start		703656
+	1021007364	catburglar-item-obtained-1	C:\ChatCambrioleur\Sounds\wwise_project\.cache\Windows\SFX\catburglar-item-obtained-1_10C4C929.wem		\Actor-Mixer Hierarchy\Default Work Unit\play-yarn-ball-pickup\catburglar-item-obtained-1		276652
```
Comment (amzn-changml) [inline Sounds/wwise/mainsoundbank.txt]: `Audio source file` appears to be pointing to `C:\ChatCambrioleur`. What happens if someone sets their project folder to a different path? Should this be a relative path?

File: Sounds/wwise_project/ChatCambrioleur.chgalvan.validationcache [hunk 2]
```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ValidationCache Version="1" SchemaVersion="103">
+	<ValidationCacheItem File="C:\ChatCambrioleur\Sounds\wwise_project\Virtual Acoustics\Factory Acoustic Textures.wwu" DateLow="1639586020" DateHigh="0"/>
```
Comment (amzn-changml) [inline Sounds/wwise_project/ChatCambrioleur.chgalvan.validationcache]: Same as the other audio source files. Is this a hardcoded path?

Comment (cgalvan) [reply Sounds/wwise_project/ChatCambrioleur.chgalvan.validationcache]: This shouldn't have been checked in, I'll delete it

File: Sounds/wwise/mainsoundbank.txt (hunk 1 above)
Comment (cgalvan) [reply Sounds/wwise/mainsoundbank.txt]: Same thing here, will delete thanks!

**Review by amzn-changml — APPROVED** [review]

**Pull request merged**